src/every.js

```diff
@@ -0,0 +1,10 @@
+export default function every(values, test) {
+  if (typeof test !== "function") throw new TypeError("test is not a function");
+  let index = -1;
```

Comment:
out of curiosity why start at -1 with ++index rather than 0 with index++ ? would save a byte :)

Reply:
Out of habit, mostly. At one point I was told it was faster…